modules/test/playwright/tests/commerce/commerceChannelDefaults.spec.ts

```diff
@@ -341,3 +341,98 @@ test('LPD-26142 A Sales Agent can manage channel defaults', async ({
 		await performLogin(page, 'test');
 	}
 });
+
+test('LPD-30466 Verify users without edit permission cannot click on channel name link', async ({
+	apiHelpers,
+	applicationsMenuPage,
+	page,
+}) => {
+	const companyId = await page.evaluate(() => {
+		return Liferay.ThemeDisplay.getCompanyId();
+	});
+
+	const site = await apiHelpers.headlessSite.createSite({
+		name: getRandomString(),
+	});
+
+	apiHelpers.data.push({id: site.id, type: 'site'});
+
+	const channel = await apiHelpers.headlessCommerceAdminChannel.postChannel({
+		siteGroupId: site.id,
+	});
+
+	apiHelpers.data.push({id: channel.id, type: 'channel'});
+
+	const user =
+		await apiHelpers.headlessAdminUser.getUserAccountByEmailAddress(
+			'demo.unprivileged@liferay.com'
+		);
+
+	const role = await apiHelpers.headlessAdminUser.postRole({
+		name: 'User' + getRandomString(),
+		rolePermissions: [
+			{
+				actionIds: ['ACCESS_IN_CONTROL_PANEL'],
+				primaryKey: companyId,
+				resourceName:
+					'com_liferay_commerce_channel_web_internal_portlet_CommerceChannelsPortlet',
+				scope: 1,
+			},
+			{
+				actionIds: ['VIEW_CONTROL_PANEL'],
+				primaryKey: companyId,
+				resourceName: '90',
+				scope: 1,
+			},
+			{
+				actionIds: ['VIEW_COMMERCE_CHANNELS'],
+				primaryKey: companyId,
+				resourceName: 'com.liferay.commerce.channel',
+				scope: 1,
+			},
+			{
+				actionIds: ['VIEW'],
+				primaryKey: companyId,
+				resourceName:
+					'com.liferay.commerce.product.model.CommerceChannel',
+				scope: 1,
+			},
+		],
+	});
+
+	await apiHelpers.headlessAdminUser.assignUserToRole(role.name, user.id);
+
+	const account = await apiHelpers.headlessAdminUser.postAccount({
+		name: getRandomString(),
+		type: 'business',
+	});
+
+	apiHelpers.data.push({id: account.id, type: 'account'});
+
+	await apiHelpers.headlessAdminUser.assignUserToAccountByEmailAddress(
+		account.id,
+		[user.emailAddress]
+	);
+
+	await performLogout(page);
+
+	await performLogin(page, user.alternateName);
+
+	try {
+		await applicationsMenuPage.goToCommerceChannels();
```

Comment:
please use the CommerceAdminChannelDetailsPage to reach the page (with the goto function)